### PR TITLE
Add get native window interface

### PR DESCRIPTION
**Pull request by xiaowei-guan**
We implement video player plugin with hole solution, it need engine provides get native window interface. we have to return to two types of windows(Elm window, Ecore_Wl2_window) for current status.
I think a good soluton would be to return only one type of window(Elm window), and I am trying to create elm window as root window and get ecore_wl2_window from elm window(https://github.com/flutter-tizen/engine/issues/167)

### CODE REVIEW

File: shell/platform/tizen/public/flutter_tizen.h [hunk 1]
```diff
@@ -17,6 +17,10 @@
 extern "C" {
 #endif
 
+// Opaque reference to a Flutter window.
+struct FlutterDesktopView;
+typedef struct FlutterDesktopView* FlutterDesktopViewRef;
```
Comment (swift-kim) [inline shell/platform/tizen/public/flutter_tizen.h:22]: TizenRenderer = View = Window = RenderTarget?

File: shell/platform/tizen/public/flutter_tizen.h [hunk 2]
```diff
@@ -91,6 +95,18 @@ FlutterDesktopGetPluginRegistrar(FlutterDesktopEngineRef engine,
...
+/**
+ * Return backing window for manipulation in host application.
+ *
+ * If run engine on a wearable device, need cast void* to Evas_Objct*,
+ * otherwise cast void* to Ecore_Wl2_Window*.
+ */
+FLUTTER_EXPORT void* FlutterDesktopViewGetWindow(FlutterDesktopViewRef view);
+
+// Returns the view associated with this registrar's engine instance.
+FLUTTER_EXPORT FlutterDesktopViewRef FlutterDesktopPluginRegistrarGetView(
```
Comment (swift-kim) [inline shell/platform/tizen/public/flutter_tizen.h:107]: Do we need the translation from View to Window? It seems just okay to get Window directly from the registrar.

File: shell/platform/tizen/tizen_renderer.h [hunk 3]
```diff
@@ -29,6 +29,7 @@ class TizenRenderer {
 
   bool IsValid() { return is_valid_; }
 
+  virtual void* GetWindow() = 0;
```
Comment (swift-kim) [inline shell/platform/tizen/tizen_renderer.h:32]: I just realized that there's already `GetWindowHandle`.

Comment (xiaowei-guan) [reply shell/platform/tizen/tizen_renderer.h:32]: yes, but EvasGL renderer return nullptr.
Hi,@bwikbs do we could  return evas_window_ by GetWindowHandle?

File: shell/platform/tizen/public/flutter_tizen.h (hunk 2 above)
Comment (xiaowei-guan) [reply shell/platform/tizen/public/flutter_tizen.h]: Good suggestion, for currently status, we can get window directly form the registrar. Thank you~

File: shell/platform/tizen/tizen_renderer.h (hunk 3 above)
Comment (bwikbs) [reply shell/platform/tizen/tizen_renderer.h:32]: @xiaowei-guan 
Sorry for late reply!
> Hi,@bwikbs do we could return evas_window_ by GetWindowHandle?

Of course it is possible! :smil

Comment (xiaowei-guan) [reply shell/platform/tizen/tizen_renderer.h]: ok,Thank you~

**Review by swift-kim — APPROVED** [review]

**Review by bwikbs — APPROVED** [review]
I guess platform view should also use this!

**Pull request merged**